apps/edxapp/vars/all/main.yml

```diff
@@ -69,14 +69,23 @@ edxapp_nginx_lms_port: 8071
 edxapp_nginx_htpasswd_secret_name: "edxapp-htpasswd"
 
 # -- celery/redis
-edxapp_celery_lms_high_priority_queue: "edx.lms.core.high"
-edxapp_celery_lms_default_priority_queue: "edx.lms.core.default"
-edxapp_celery_lms_low_priority_queue: "edx.lms.core.low"
-edxapp_celery_lms_high_mem_queue: "edx.lms.core.high_mem"
+#
+# To avoid deploying to much ressouces when it is not required, all queues
```

Comment:
s/to much ressources/several celery workers
(we will only deploy 1 then! It's better than "not too many"))